# HG changeset patch
# Date 1673190532 -7200
#      Sun Jan 08 17:08:52 2023 +0200
# Branch py3.8
# Node ID d8e6b37f5231c7b59b2b639a651dd696a3732cc3
# Parent  c758c7d7097f60adffc8d9e8a8fc0dea18902e6f
make code compatible with conda

diff -r c758c7d7097f -r d8e6b37f5231 lib_pypy/_ssl_build.py
--- a/lib_pypy/_ssl_build.py	Fri Jan 06 00:00:39 2023 +0200
+++ b/lib_pypy/_ssl_build.py	Sun Jan 08 17:08:52 2023 +0200
@@ -1,3 +1,4 @@
+import os
 import sys
 from _cffi_ssl import _cffi_src
 sys.modules['_cffi_src'] = _cffi_src
@@ -60,7 +61,7 @@
 
 if __name__ == '__main__':
     ffi.compile(verbose=True)
-    if sys.platform == 'win32':
+    if sys.platform == 'win32' and "PYPY_PACKAGE_NO_DLLS" not in os.environ:
         # copy dlls from externals to the pwd
         # maybe we should link to libraries instead of the dlls
         # to avoid this mess
